src/style.css

```diff
@@ -1,3 +1,113 @@
+html {
+  font-family: Roboto, sans-serif;
+}
+
 body {
   margin: 0;
+  padding-top: 1px;
```

Comment:
Be consistent with your vertical margins. Add only top or only bottom margin, but not both. In this case, you have a 'padding-top' on the 'body' which can potentially collapse with margins of elements inside it.